assets/js/components/ClusterDetails/ChecksSelectionGroup.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+import { Disclosure, Switch, Transition } from '@headlessui/react';
+
+import { EOS_KEYBOARD_ARROW_RIGHT } from 'eos-icons-react';
+
+export const groupState = {
+  None: 0,
+  Some: 1,
+  All: 2,
+};
```

Comment:
You could also do

```js
export const SOME = 'SOME';

// and then
import { ALL, SOME, NONE } from './ChecksSelectionGroup';
```

Then you could also use the constants this way:

```js
const switchClasses = {
  [NONE]: 'bg-gray-200',
  [SOME]: 'bg-green-300',
  [ALL]: 'bg-jungle-green-500',
};
```

Reply:
Yep, gone this way.
Basically, because it is much better to find compile time errors, as the "enum" dict can get anything after the dot, creating some potential issues.